clients/admin-ui/src/features/common/table/v2/filters/GlobalFilterV2.tsx

```diff
@@ -23,16 +23,16 @@ export const GlobalFilterV2 = ({
     [setGlobalFilter],
   );
 
-  const onClear = () => {
+  const onClear = useCallback(() => {
     setValue(undefined);
     setGlobalFilter(undefined);
-  };
+  }, [setValue, setGlobalFilter]);
```

Comment:
Great. This is the correct fix for the error. However, it may have unintentional consecuences if the setGlobalFilter prop is changing. Can you search for setGlobalFilter={ and add a useCallback wrapper if the prop is changing? Most of them are useState hooks so it should be fine, but some of them are function definitions inside the component without the wrapper.

Reply:
Thanks for catching this, fixed.